PostHog/UIViewController.swift

```diff
@@ -58,11 +58,17 @@
             }
         }
 
-        @objc func viewDidApperOverride(animated: Bool) {
+        @objc func viewDidAppearOverride(animated: Bool) {
+            // ignore views from keyboard window
+            // these may include: UIInputWindowController, _UICursorAccessoryViewController, UICompatibilityInputViewController,UIKeyboardHiddenViewController_Autofill and others
+            if let window = viewIfLoaded?.window, window.isKeyboardWindow {
+                return
+            }
+
             captureScreenView(viewIfLoaded?.window)
             // it looks like we're calling ourselves, but we're actually
             // calling the original implementation of viewDidAppear since it's been swizzled.
-            viewDidApperOverride(animated: animated)
+            viewDidAppearOverride(animated: animated)
```

Comment:
I wonder if we should not capture `captureScreenView` here but still forward the call to `viewDidAppearOverride`.
Feels like the right thing to do otherwise we are breaking an OS/apps behaviour

Reply:
we can also do `let window = viewIfLoaded?.window` so we dont do the `?.` twice in the same object when calling `captureScreenView`